sql_prep/models/achievements/agg__lepsi_nez_prumer.sql

```diff
@@ -1,29 +1,20 @@
-/*SELECT nazev, clovek_id, COUNT() FROM {{ ref('clovek_debata') }}
-WHERE nazev IN ('Debatní liga XXVIII.', 'Debate League XXVIII.')
-AND kidy > 75
-GROUP BY nazev, clovek_id*/
-
 WITH achievement AS (
     SELECT
         clovek_id,
-        rocnik,
+        school_year,
         COUNT() AS pocet_debat,
-        /*CASE
-            WHEN kidy > 75 THEN 'Lepší než průměr' 
-            ELSE null
-        END AS achievement_text,*/
-    FROM {{ ref('clovek_debata') }}
-    WHERE clovek_id IS NOT null AND kidy > 75
-    GROUP BY clovek_id, rocnik
+    FROM {{ ref('base__debater_debata') }}
+    WHERE kidy > 75
+    GROUP BY clovek_id, school_year
 ),
 
 final AS(
     SELECT
         clovek_id,
-        rocnik,
+        school_year,
         'Lepší než průměr' AS achievement_name,
         'Gratuluji, povedlo se ti být' || 'achievement_text'|| 'v' || pocet_debat || 'debatách za tento rok!' AS achievement_description,
```

Comment:
This will result in a string like this: "Gratuluji, povedlo se ti býtachievement_textv1debatách za tento rok!" - fix the spaces and maybe remove the achievement_text thing?

Reply:
fixed